org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/editor/quickfix/DiagnosticResolutionAcceptor.java

```diff
@@ -35,12 +35,16 @@ public DiagnosticResolutionAcceptor(DiagnosticModificationContext.Factory modifi
 	}
 
 	public void accept(String label, IModification<EObject> modification) {
-		issues.add(new DiagnosticResolution(label, modificationContextFactory, modification));
+		resolutions.add(new DiagnosticResolution(label, modificationContextFactory, modification));
+	}
+
+	public void accept(final String label, final ITextModification modification) {
```

Comment:
Nit: final parameters are not adding value here. Not actionable.

You may want to use Oomph to setup your Xtext dev environment, to simplify the review process for you and the reviewer though :)

Reply:
Sorry for that. I did that this morning already to write the test, it was just too error-prone to do that in GitHub editor :) It will be part of the next push.